test/media-player.test.js

```diff
@@ -1,5 +1,5 @@
 import '../media-player.js';
-import { expect, fixture, html } from '@open-wc/testing';
+import { expect, fixture, html } from '@brightspace-ui/testing';
 import { runConstructor } from '@brightspace-ui/core/tools/constructor-test-helper.js';
```

Comment:
This can be retrieved from the testing library now:
```suggestion
import { expect, fixture, html, runConstructor } from '@brightspace-ui/testing';
```